.travis.yml

```diff
@@ -1,28 +1,22 @@
 sudo: required
 dist: trusty
-# Force travis to use its minimal image with default Python settings
 language: generic
-compiler:
-  - gcc
-
+notifications:
+  email:
+    on_success: change
+    on_failure: always
 env:
   global:
-    - CATKIN_WS=~/catkin_ws
-    - CATKIN_WS_SRC=${CATKIN_WS}/src
-    - CATKIN_WS_UNDERLAY=~/catkin_ws_underlay
-    - CATKIN_WS_UNDERLAY_SRC=${CATKIN_WS_UNDERLAY}/src
-    - CI_ROS_DISTRO="indigo"
-    - DEFAULT_ROSINSTALL="https://raw.githubusercontent.com/ipa320/care-o-bot/${CI_ROS_DISTRO}_dev/.travis.rosinstall"
-
+    - UPSTREAM_WORKSPACE=file
+  matrix:
+    - ROS_DISTRO="indigo" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
+    - ROS_DISTRO="jade" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
+    - ROS_DISTRO="kinetic" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
+matrix:
+  allow_failures:
+    - env: ROS_DISTRO="jade" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu UPSTREAM_WORKSPACE=file
 install:
-  - mkdir -p $CATKIN_WS_UNDERLAY_SRC
-  - git clone https://github.com/ipa320/cob_robots.git /tmp/cob_robots
-  - mv /tmp/cob_robots/cob_supported_robots  $CATKIN_WS_UNDERLAY_SRC/
-  - rm -rf /tmp/cob_robots
-  - wget https://raw.githubusercontent.com/ipa320/care-o-bot/${CI_ROS_DISTRO}_dev/.travis.install.sh
-  - bash .travis.install.sh
-
+  - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config
 script:
-  - wget https://raw.githubusercontent.com/ipa320/care-o-bot/${CI_ROS_DISTRO}_dev/.travis.script.sh
-  - bash .travis.script.sh
-  - set -e; for x in $(find . -name '*.xacro'); do echo "Testing $x"; $CATKIN_WS/install/env.sh rosrun xacro xacro "$x" > /dev/null; done
```

Comment:
Hint to myself: https://github.com/ros-industrial/industrial_ci/blob/master/doc/index.rst#optional-environment-variables